config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AdvancedXeninol-D0JaazeIQ8qctaMx9xTqEw==.json

```diff
@@ -0,0 +1,80 @@
+{
+  "questIDHigh:4": 1099540675571499978,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": -2396217196736196889,
+      "questIDLow:4": -8084303837458271711
+    },
+    "1:10": {
+      "questIDHigh:4": 934917285194189830,
+      "questIDLow:4": -4721119019196807237
+    },
+    "2:10": {
+      "questIDHigh:4": 4807853113064639940,
+      "questIDLow:4": -5802810217373252190
+    }
+  },
+  "questIDLow:4": -7154632998014096877,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "miscutils:itemHeavyPlateAdvancedNitinol",
+        "Count:3": 1,
+        "Damage:2": 0,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Advanced Xeninol",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "It\u0027s here that you might yourself with some useful stuff alongside power. Xenon will soon be useful to use in the EBF, and Advanced Nitinol? If you don\u0027t already have a Quantum Anomaly setup for it, you can use this. Who knows what\u0027ll come in the future?"
```

Comment:
will soon be -> is (ebf change is already merged)

Reply:
Yeah, I was working on this before the merge happened, thanks for letting me know.